lib/registrant.rb

```diff
@@ -0,0 +1,25 @@
+require './lib/registrant'
+
```

Comment:
Here, we would want to create the Registrant class something like:
```
class Registrant

   attr_reader :name, :age, :permit
 
    def initialize(name, age, permit = false)
          @name = name
          @age = age
          @permit = permit
     end
.... more methods below
```

Be careful of copying and pasting interaction pattern code. That is code that you can use to create tests in order to build out your functionality, but it shouldn't got straight into class files or spec files